atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -59,7 +59,19 @@ private CassandraKeyValueServices() {
         // Utility class
     }
 
-    static void waitForSchemaVersions(Cassandra.Client client, String tableName, int schemaTimeoutMillis)
+    static void waitForSchemaVersions(
+            Cassandra.Client client,
+            String tableName,
+            int schemaTimeoutMillis)
+            throws TException {
+        waitForSchemaVersions(client, tableName, schemaTimeoutMillis, false);
```

Comment:
Agreed with @clockfort - definitely worth a separate PR and some extra thinking, and also, defaulting to false is the right way to go here.